From ae5ed018f97c78e9a23eac8b83fcc6b498e2358f Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Tue, 17 Nov 2020 11:21:25 -0500
Subject: [PATCH] No python3.exe on Windows

---
 tests/test_ansi2html.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_ansi2html.py b/tests/test_ansi2html.py
index d725097..e8138d4 100644
--- a/tests/test_ansi2html.py
+++ b/tests/test_ansi2html.py
@@ -471,5 +471,5 @@ class TestAnsi2HTML:
         assert result.returncode == 0
 
     def test_command_module(self) -> None:
-        result = run(["python3", "-m", "ansi2html", "--version"], check=True)
+        result = run(["python", "-m", "ansi2html", "--version"], check=True)
         assert result.returncode == 0
-- 
2.33.1

